From 19304f492e25b22e7c01ac41819ba8996c4384d3 Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Tue, 15 May 2018 15:01:07 +0800
Subject: [PATCH 265/706] drm: hdp: emu: Reduce the time for HPD on Emulator

This patch is to reduce the time for HPD on emulator.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit 0903328ddb2c5ed2b0e511a5e2e4a0e40b5f60ed)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp.c b/drivers/gpu/drm/imx/hdp/imx-hdp.c
index 9f0cf086f3a1..0ecc42940bf4 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp.c
@@ -851,7 +851,7 @@ static int hpd_det_worker(void *_dp)
 		} else if (evt & 0xf)
 			printk(KERN_DEBUG "evt=0x%x\n", evt);
 
-		schedule_timeout_idle(200);
+		schedule_timeout_idle(1);
 	}
 
 	return 0;
-- 
2.17.1

